rts/Sim/Projectiles/ProjectileMemPool.h

```diff
@@ -9,10 +9,11 @@
 
 #include "Sim/Projectiles/WeaponProjectiles/StarburstProjectile.h"
 
-static constexpr size_t PMP_S = AlignUp(sizeof(CStarburstProjectile), 4); //biggest in size
+static constexpr size_t PMP_ALIGN = 8;
```

Comment:
why 8? used to be 4

Reply:
`sizeof(CStarburstProjectile)` is 1048, which mod 8 is 0: It was already aligned to 8. Specifying that alignment is supposed to be 4, failed the static_assert check in alloc, because many types aligned in this memory pool required alignment to 8.

Reply:
alright, consider
```suggestion
static constexpr size_t PMP_ALIGN = 8; // smallest that fits the needs of all the various projectile types
```